templates/base/head_navbar.tmpl

```diff
@@ -128,7 +128,7 @@
 					</a>
 					{{end}}
 					<div class="divider"></div>
-					<a class="item link-action" href data-url="{{AppSubUrl}}/user/logout">
+					<a class="item{{if not .ReverseProxyLogoutRedirect}} link-action" href="" data-url={{else}}" href={{end}}"{{AppSubUrl}}/user/logout">
```

Comment:
Does anything break if we remove these empty `href` attributes? If not, remove them.